examples/controls_example/public/react_controls/control_panel.tsx

```diff
@@ -186,3 +212,49 @@ export const ControlPanel = <ApiType extends DefaultControlApi = DefaultControlA
     </EuiFlexItem>
   );
 };
+
+/**
```

Comment:
How about putting ControlClone into its own file?

Reply:
Done in 1d0b5d28b8c6942d247bed6bcbcf98f0292c9bed